### PR TITLE
Adding 2 new arts

### PR DESCRIPTION
**Pull request by CodesOfRishi**
Added 2 new ASCII arts named,
- linuxrules
- redhat

Files containing both the ASCII art resides in `ascii_arts/` directory.

### CODE REVIEW

**Comment by BeyondMagic** [comment]
Thank you so much!!!

**Comment by BeyondMagic** [comment]
Actually, can you me confirm to me that `linuxrules` is yours or from `hjm`? If the latter, the first comment is designed to credit the original creator and authors who modify the art.

*Of course there's no copyright behind ASCII arts, but I would like to follow the [respect ASCII artists campaign](https://asciiartist.com/respect-ascii-artists-campaign/) for kindness.*

**Comment by CodesOfRishi** [comment]
Yes, it's not mine, I got both of the arts from ASCII Art Archive website.
I wasn't aware of the [respect ASCII artists campaign](https://asciiartist.com/respect-ascii-artists-campaign/).
Please, do inform me if I did anything that I shouldn't have done or anything else I should take care of from here onwards.

If you say, I can create another PR crediting the source(s) or artist(s).

Thanks

**Comment by BeyondMagic** [comment]
I understand, the campaign is listed on README.md as an image, but I should've have made that clear on the [adding arts section](https://github.com/ninecath/nincat/#%EF%B8%8F--adding-new-ascii-arts).

Anyways, thanks for the contributions for now, I will revert the changes of this PR so that you can add a new PR linking to the original authors as I don't have the names besides the first one from *hjm*.

---

As for creating new arts, I understand that it's difficult now since writing ASCII art on terminal has always been tiresome, so I'm been working on this issue separately #1.

File: ascii_arts/linuxrules 16 37 [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+     dZP        qKRb
+    dZP          qKKb
+   fZP            SMMb
+   HZM            MMMM
+   FqM            MMMM
+ __| ".        |\dS"qML
+ |    `.       | `' \Zq
+_)      \.___.,|     .'
+\____   )MMMMMM|   .'
+     `-'       `--' hjm
```
Comment (BeyondMagic) [inline ascii_arts/linuxrules 16 37:17]: Here, you could remove his/her artist name (hjm) and put it at the first line.

**Pull request merged**